### ISSUE TITLE
How can we stop shuffling in the dataset?

### ISSUE THREAD

**Issue by ajeetksingh** [facebook/fb.resnet.torch#55]
Hi,

It seems that, there is some sort of shuffling in the dataset, before the
training starts.
Is there any way I can disable the shuffling?

Thanks.

**Comment by colesbury** [comment]
https://github.com/facebook/fb.resnet.torch/blob/65e675a8cc7c96918d140656af55941afbfa8104/dataloader.lua#L65
